### ISSUE TITLE
Analysis of Bump direction

### ISSUE THREAD

**Issue by CeleritasCelery** [fitzgen/bumpalo#243]
As part of the work I was doing on [minimum alignment](https://github.com/fitzgen/bumpalo/pull/242), I wrote a [blog post](https://coredumped.dev/2024/03/25/bump-allocation-up-or-down/) analyzing bump allocator performance. I wanted to share that here to see if you had any insights or feedback since this is the most popular Rust bump allocation crate.

**Comment by fitzgen** [comment]
Thanks for writing that up and sharing, I did actually see that post late last night on r/rust, but I haven't had a chance to do more than skim it thus far. I'm a bit swamped at the moment, so I probably won't be able to get to do a more in-depth analysis for a week or so. But again I appreciate you sharing it!

**Comment by VorpalBlade** [comment]
I'm curious as to the result of this (it has been far more than a week). Is it likely this might lead to changes in bumpalo in the future?

**Comment by fitzgen** [comment]
Yes, I have some changes, including some perf improvements, in a branch but it isn't ready for merging yet and I haven't had time to clean things up yet.
